sdv/data_processing/data_processor.py

```diff
@@ -587,13 +587,24 @@ def _create_config(self, data, columns_created_by_constraints):
 
             elif sdtype == 'unknown':
                 sdtypes[column] = 'pii'
-                transformers[column] = AnonymizedFaker(
-                    function_name='bothify',
-                )
-                transformers[column].function_kwargs = {
+                function_name = 'bothify'
+                function_kwargs = {
                     'text': 'sdv-pii-?????',
                     'letters': '0123456789abcdefghijklmnopqrstuvwxyz',
                 }
+                if pd.api.types.is_numeric_dtype(data[column]):
+                    digits = data[column].apply(lambda x: len(str(abs(x))))
+                    max_digits = max(digits)
+                    min_digits = min(digits)
```

Comment:
instead of applying to the whole column, why don't we just get the min and max of the column and only compute the digits for those? I think the apply function is kind of slow especially when it has to convert everything to strings